services/src/handlers/layers.rs

```diff
@@ -171,6 +266,168 @@ async fn list_collection_handler<C: Context>(
+                      "4": [
+                        159,
+                        225,
+                        127,
+                        255
+                      ],
+                      "5": [
+                        125,
+                        194,
+                        127,
+                        255
+                      ],
+                      "6": [
+                        195,
+                        127,
+                        126,
+                        255
+                      ],
+                      "7": [
+                        188,
+                        221,
+                        190,
+                        255
+                      ],
+                      "8": [
+                        224,
+                        223,
+                        133,
+                        255
+                      ],
+                      "9": [
+                        226,
+                        221,
+                        7,
+                        255
+                      ],
+                      "10": [
+                        223,
+                        192,
+                        125,
+                        255
+                      ],
+                      "11": [
+                        66,
+                        128,
+                        189,
+                        255
+                      ],
+                      "12": [
+                        225,
+                        222,
+                        127,
+                        255
+                      ],
+                      "13": [
+                        253,
+                        2,
+                        0,
+                        255
+                      ],
+                      "14": [
+                        162,
+                        159,
+                        66,
+                        255
+                      ],
+                      "15": [
+                        255,
+                        255,
+                        255,
+                        255
+                      ],
+                      "16": [
+                        192,
+                        192,
+                        192,
+                        255
+                      ]
+                    },
+                    "noDataColor": [
+                      0,
+                      0,
+                      0,
+                      0
+                    ],
+                    "defaultColor": [
+                      0,
+                      0,
+                      0,
+                      0
+                    ]
+                  }
+                },
+                "properties": [],
+                "metadata": {}
+              })
+        )
+    ),
+    params(
+        ("provider" = DataProviderId, description = "Data provider id"),
+        ("layer" = LayerCollectionId, description = "Layer id"),
+    ),
+    security(
+        ("session_token" = [])
+    )
+)]
```

Comment:
Maybe you can make sense of this error

![grafik](https://user-images.githubusercontent.com/4947061/193007874-eb6ead92-c93c-41a2-b9bb-500f0ea1e2ec.png)

Reply:
you switched data and layer provider id. it has to be the other way around.